static/app/components/charts/eventsRequest.tsx

```diff
@@ -395,9 +400,15 @@ class EventsRequest extends React.PureComponent<EventsRequestProps, EventsReques
       // Convert the timeseries data into a multi-series result set.
       // As the server will have replied with a map like:
       // {[titleString: string]: EventsStats}
+      let timeframe: {start: string; end: string} | undefined = undefined;
       const results: MultiSeriesResults = Object.keys(timeseriesData)
+        .filter((seriesName: string) => !['start', 'end'].includes(seriesName))
```

Comment:
Is this still necessary now that the start/end is attached to each individual response?

Reply:
👍  no longer necessary